explorer/app/components/Project/components/common/projectMapper.ts

```diff
@@ -0,0 +1,12 @@
+// App dependencies
+import { ProjectResponse } from "../../../../models/responses";
+
+/**
+ * Maps project description.
+ * @param project
+ */
+export function mapProjectDescription(
```

Comment:
What do you think about keeping the naming convention simple here and call this `getProjectDescription`?